fuel/app/classes/controller/api/asset.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Materia
+ * License outlined in licenses folder
+ */
+
+use \Materia\Perm;
+use \Materia\Perm_Manager;
+
```

Comment:
Getting the error "Class 'Msg' not found" when user is not logged in. We should import \Materia\Msg here.